test/com/google/cose/Encrypt0MessageTest.java

```diff
@@ -0,0 +1,46 @@
+package com.google.cose;
+
+import co.nstant.in.cbor.model.ByteString;
+import co.nstant.in.cbor.model.DataItem;
+import co.nstant.in.cbor.model.Map;
+import co.nstant.in.cbor.model.UnsignedInteger;
+import com.google.cose.utils.CborUtils;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class Encrypt0MessageTest {
+  @Test
+  public void testDeserialize() {
+    Encrypt0Message message = Encrypt0Message.deserialize(TestUtilities.hexStringToByteArray(
+      "8343A10101A1054C02D1F7E6F26C43D4868D87CE582460973A94BB2898009EE52ECFD9AB1DD25867374B162E2C0"
+          + "3568B41F57C3CC16F9166250B"
+    ));
+    Assert.assertEquals("60973A94BB2898009EE52ECFD9AB1DD25867374B162E2C03568B41F57C3CC16F9166250B",
+        TestUtilities.bytesToHexString(message.getCiphertext()));
+    Assert.assertEquals("A10101",
```

Comment:
I assume this is actually a `Map` now in the latest revision (per Max's feedback above)?

Reply:
yes. But we had a conversation about keeping both header bytes as well as headers for now.

Reply:
Ah ok. SGTM.